compiler/renderer.nim

```diff
@@ -976,6 +976,11 @@ proc infixArgument(g: var TSrcGen, n: PNode, i: int) =
   if needsParenthesis:
     put(g, tkParRi, ")")
 
+proc isCustomLit(n: PNode): bool =
```

Comment:
this needs to be exposed in some API in macros.nim

Reply:
Can be done later, when the need arises.